### ISSUE TITLE
Typos in docs/guide/event_system.rst

### ISSUE THREAD

**Issue by A03ki** [flexxui/flexx#630]
[https://github.com/flexxui/flexx/blob/master/docs/guide/event_system.rst](https://github.com/flexxui/flexx/blob/master/docs/guide/event_system.rst)

In [Properties represent state](https://github.com/flexxui/flexx/blob/master/docs/guide/event_system.rst#properties-represent-state), a component is defined.

```python
class MyObject(flx.Component):

    foo = flx.AnyProp(8, settable=True, doc='can have any value')
    bar = flx.IntProp()
```
The next code is this.

```python
c = MyComponent(foo=42)
```

It can't create this component because "MyComponent" is not defined.

So, the name of class object should change "MyObject" to "MyComponent" because "MyObject"  is defined in [The Component class](https://github.com/flexxui/flexx/blob/master/docs/guide/event_system.rst#the-component-class).

```python
class MyComponent(flx.Component):

    foo = flx.AnyProp(8, settable=True, doc='can have any value')
    bar = flx.IntProp()
```

```python
c = MyComponent(foo=42)
```
It went well!

---

>In the example below, the label text will be automatically updated when the username property changes:
>
>flx.Label(flex=1, text=lambda: 'count is ' + str(self.counter))
> [Properties represent state](https://github.com/flexxui/flexx/blob/master/docs/guide/event_system.rst#properties-represent-state)

I think username the property is a typo, so it should change "username" to "counter".

---

And, descrive❌ describe⭕️ in [The Component class](https://github.com/flexxui/flexx/blob/master/docs/guide/event_system.rst#the-component-class).

Please let me know if there are any problems.

**Comment by almarklein** [comment]
Fixed in #631, "descrive" appears to have been already fixed (but the online docs don't yet include the fix).

**Comment by A03ki** [comment]
Thanks!
